app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -35,6 +35,38 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_a_heading_on_one_checkbox:
+    description: One checkbox does not require a fieldset and therefore does not require a legend. However, if a heading is supplied, a fieldset will be included in the component and the heading used as the legend, as shown.
```

Comment:
This is still a bit contradictory to the usage of the checkbox from above: _You must pass a heading when using this component_

If that's the case then the case of one checkbox will always be rendered with a fieldset and legend?

Reply:
Good point. I've updated the documentation by removing this bit - I think the examples explain themselves better now anyway.